### PR TITLE
Fix:Image garbage collection is always disabled in kubelet in 1.11 

### LINKED ISSUE #66046: Image garbage collection is always disabled in kubelet in 1.11
In the `StartGarbageCollection` function, there is a `defer close(stopChan)` that always runs and closes the channel when returning. This means that instead of running periodically, the image garbage collection will be terminated immediately when the function returns. 

https://github.com/kubernetes/kubernetes/blob/91e7b4fd31fcd3d5f436da26c980becec37ceefe/pkg/kubelet/kubelet.go#L1247-L1253

The bug was introduced by #51423 in 1.11

/cc @derekwaynecarr @jiaxuanzhou @kubernetes/sig-node-bugs @dashpole

### PR DESCRIPTION
**Pull request by jiaxuanzhou**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes  #66046

**Special notes for your reviewer**:
@yujuhong @derekwaynecarr 
**Release note**:

```release-note
Fix the bug where image garbage collection is disabled by mistake.
```

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1239,11 +1239,10 @@ func (kl *Kubelet) StartGarbageCollection() {
 	}, ContainerGCPeriod, wait.NeverStop)
 
 	stopChan := make(chan struct{})
-	defer close(stopChan)
 	// when the high threshold is set to 100, stub the image GC manager
 	if kl.kubeletConfiguration.ImageGCHighThresholdPercent == 100 {
 		glog.V(2).Infof("ImageGCHighThresholdPercent is set 100, Disable image GC")
-		go func() { stopChan <- struct{}{} }()
+		go func() { close(stopChan) }()
```
Comment (yujuhong) [inline pkg/kubelet/kubelet.go]: Why not just returning here?

**Comment by yujuhong** [comment]
Please add a release note.

**Comment by yujuhong** [comment]
/cc @foxish, the 1.11 patch release manager.
This should be included in 1.11.1

**Comment by yujuhong** [comment]
/ok-to-test

@dashpole, do we have any image gc tests?

File: pkg/kubelet/kubelet.go (hunk 1 above)
Comment (jiaxuanzhou) [reply pkg/kubelet/kubelet.go]: modified

**Comment by foxish** [comment]
ack, let's cherrypicks this into 1.11.1

**Comment by yujuhong** [comment]
@jianglingxia cold you please create PRs to cherrypick the fix to 1.11? Thanks.

**Pull request merged**